### ISSUE TITLE
oceQnet zero for 2012-10-28T00:00:00

### ISSUE THREAD

**Issue by antonimmo** [MITgcm/xmitgcm#212]
At the beginning I thought it was a mistake on my side, as I temporarily store hourly snapshots of the variables I need in order to avoid repeated calls to the API. Then I realised it is something else, so I went to ECCO site expecting to find a zero-size oceQnet file [for that particular timestamp](https://data.nas.nasa.gov/ecco/data.php?dir=/eccodata/llc_4320/compressed/0001430784), but it doesn't seem to be the case.

As a demo, I prepared this code snippet

```python
from xmitgcm import llcreader
import matplotlib.pyplot as plt

model = llcreader.ECCOPortalLLC4320Model()
model.iter_stop = 1495152 # until 2012-11-15T14:00:00

ds = model.get_dataset(type='faces')
sel_ds = ds.isel(time=9864,k=0,face=2)  # 2012-10-28T00:00:00
sel_ds.time.values

plt.figure(figsize=(12,10))
sel_ds.oceQnet.plot()
plt.show()
```

from which I get

![oceqnet_t9864](https://user-images.githubusercontent.com/6156024/87337645-a22ccd00-c4f8-11ea-8c7d-ca67230938e2.png)

If I run that for the next snapshot (2012-10-28T01:00:00), I get a completely different result

![oceqnet_t9865](https://user-images.githubusercontent.com/6156024/87337840-f768de80-c4f8-11ea-84b4-60785362c3a5.png)

I'm pretty sure this issue comes from the data itself, but I'd like to get your opinion first.

As a side note, I found out the variable description _"net surface heat flux into the ocean (+=down), >0 increases theta"_
is inaccurate as the sign seems to be inverted. One would expect Theta to decrease around 1 am, while the plot shows oceQnet>0. However, it might be better to create a separate issue for that.

Thanks.

**Comment by rabernat** [comment]
Hi @antonimmo. Thanks for your issue.

My best guess is that the data for that date is indeed blank. If the data is not available, an error is raised. If you want to verify this, you could download the data manually from the ecco data portal.

**Comment by antonimmo** [comment]
If that were the case, should I raise an issue on MITgcm repo instead?

**Comment by rabernat** [comment]
No, they do not maintain the data. If you think you discovered a problem in the data itself, you should probably email Dimitris Menemenlis.
